clang/include/clang/Driver/Options.td

```diff
@@ -931,7 +931,9 @@ def O : Joined<["-"], "O">, Group<O_Group>,
 def O_flag : Flag<["-"], "O">, Visibility<[ClangOption, CC1Option, FC1Option]>,
   Alias<O>, AliasArgs<["1"]>;
 def Ofast : Joined<["-"], "Ofast">, Group<O_Group>,
-  Visibility<[ClangOption, CC1Option, FlangOption]>;
+  Visibility<[ClangOption, CC1Option, FlangOption]>,
+  HelpText<"Deprecated; use '-O3 -ffast math' for the same behavior,"
+  " or '-O3' to enable conforming optimizations only">;
```

Comment:
```suggestion
  " or '-O3' to enable only conforming optimizations">;
```

I apologize for the late nitpick here but this feels better worded to me since the only applies to enable.

Reply:
Applied, thank you